components/organisms/common/Portal.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+type PortalProps = {
+  children: React.ReactNode;
+};
+const Portal = ({ children }: PortalProps) => {
+  const el = typeof window !== 'undefined' && document.getElementById('portal');
```

Comment:
```suggestion
  const el = typeof window !== 'undefined' && document.getElementsByTagName('body')[0];
```
로 바꾸시면 document에 portal을 정의할 필요가 없습니다!

Reply:
저는 body에 넣는거보다는 dom하나 만들어서 명시적으로 넣어주시는게 보기 좋다고 생각해요